axelrod/ipd_params.py

```diff
@@ -0,0 +1,21 @@
+"""Implements ipd_params."""
```

Comment:
Similarly, here it would probably be useful to explicitly describe the define parameters. IPD is a symmetric simultaneously play game with outcome Cooperate (C) and Defect (D). Scoring is determined by a matrix of payoff R, S, T, P ...